framework/wazuh/rbac/orm.py

```diff
@@ -2875,206 +3218,21 @@ def migrate_data(self, source: str, target: str, from_id: int = None, to_id: int
         to_id : id
             ID which the resources will be migrated to.
         """
-        old_users = self.get_data(source, User, User.id, from_id=from_id, to_id=to_id)
-        with AuthenticationManager(self.sessions[target]) as auth_manager:
-            for user in old_users:
-                if user.id in (WAZUH_USER_ID, WAZUH_WUI_USER_ID):
-                    auth_manager.update_user(user.id, user.password, hashed_password=True)
-                    continue
-                
-                status = auth_manager.add_user(username=user.username,
-                                               password=user.password,
-                                               created_at=user.created_at,
-                                               user_id=user.id,
-                                               hashed_password=True,
-                                               check_default=False)
-
-                if status is False:
-                    logger.warning(f"User {user.id} ({user.username}) is part of the new default users. "
-                                   f"Renaming it to '{user.username}_user'")
-
-                    auth_manager.add_user(username=f"{user.username}_user",
-                                          password=user.password,
-                                          created_at=user.created_at,
-                                          user_id=user.id,
-                                          hashed_password=True,
-                                          check_default=False)
-        
-        # This is to avoid an error when trying to update default users roles, policies and rules
-        if from_id == WAZUH_USER_ID and to_id == WAZUH_WUI_USER_ID:
-            return
+        list_if_must_run_resources = [AuthenticationManager]
+        list_of_resources = [RolesManager, RulesManager, PoliciesManager, UserRolesManager,
+                             RolesPoliciesManager, RolesRulesManager]
```

Comment:
```suggestion
        resources = [RolesManager, RulesManager, PoliciesManager, UserRolesManager,
                             RolesPoliciesManager, RolesRulesManager]
```
It is not necessary to generate a list just for the `AuthenticationManager` element. Also, its name had a typo and the variable name should be independent of its type.